plugins/woocommerce/tests/e2e-pw/tests/merchant/products/add-variable-product/create-product-attributes.spec.js

```diff
@@ -71,9 +71,7 @@ test.describe( 'Add product attributes', () => {
 						.click();
 
 					await expect(
-						page.getByRole( 'heading', {
-							name: 'New attribute',
-						} )
+						page.locator( '.woocommerce_attribute .attribute_name.placeholder' )
```

Comment:
Solves the first problem (which is that this use of `page.getByRole` matches more than one entity), but we get essentially the same problem a little further on. Also, it would be better to keep using `page.getByRole` if we can.

Reply:
Could also just throw a .first() or .last() on the end of the page.getByRole() for now to keep the selector as is.